__tests__/app.test.js

```diff
@@ -0,0 +1,31 @@
+const seed = require('../db/seeds/seed');
+const testData = require('../db/data/test-data/index');
+const db = require('../db/connection');
+const request = require("supertest");
+const app = require("../app")
+
+beforeEach(() => seed(testData));
+afterAll(() => db.end())
+
+describe("app test suite", () => {
+    test("200 GET all topics at endpoint of /api/topics and return array of all topic objects", () => {
+        return request(app)
+        .get("/api/topics")
+        .expect(200)
+        .then(({ body }) => {
+            expect(body.topics).toHaveLength(3);
+            body.topics.forEach((topic) => {
+                expect(topic).toHaveProperty("description", expect.any(String));
+                expect(topic).toHaveProperty("slug", expect.any(String));
+            })
+        })
+    })
+    test("404 responds with an error message when passed the wrong endpoint", () => {
+        return request(app)
+        .get("/api/banana")
+        .expect(404)
+        .then(({ body }) => {
+            expect(body.msg).toBe("Data Not Found");
+        })
+    })
```

Comment:
This error test isn't specific to this endpoint, in theory going to /api/banana is the same as going to /nonsense - it's a path that doesn't match in your server. 

I would extract this out to it's own describe block, separate from this endpoint's tests. 

Apart from that, we don't have other errors to consider 👍

Reply:
I have updated the tests suite to do this. Am not sure how i show it to you with out doing a new pull request. Are you able to see the changes as i have pushed to nc-news-2?